Tests.Integration.Presentation.Web/Properties/AssemblyInfo.cs

```diff
@@ -33,3 +34,4 @@
 // [assembly: AssemblyVersion("1.0.*")]
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: CollectionBehavior(DisableTestParallelization = true)] //Disable parallelization for tests since some tests expect to control the state of global objects.
```

Comment:
Husk at fjerne attributterne fra de enkelte test classes også. De to virker ikke i samspil så de gamle skal væk

Reply:
Skal fjernes. Problemet med parallelisering er løst i master nu :)